Makefile

```diff
@@ -399,7 +399,14 @@ dev-azure-nuke: envsubst azure-nuke ## Warning: Destructive! Nuke all Azure reso
 	@rm config/dev/azure-cloud-nuke.yaml
 
 .PHONY: cli-install
-cli-install: clusterawsadm clusterctl cloud-nuke envsubst yq awscli ## Install the necessary CLI tools for deployment, development and testing.
+cli-install: clusterawsadm clusterctl cloud-nuke envsubst yq awscli jq ## Install the necessary CLI tools for deployment, development and testing.
+	@required_bins="clusterawsadm clusterctl cloud-nuke envsubst yq aws jq"; \
+	for bin in $$required_bins; do \
+		if ! command -v $$bin >/dev/null 2>&1; then \
+			echo "Error: '$$bin' is not installed or not in your PATH. Exiting."; \
+			exit 1; \
+		fi; \
+	done
```

Comment:
I don't understand the logic behind this. 
We just installed the tools needed (they're mentioned in the dependencies of the target) in the local bin directory and then we're testing that they're present in the PATH (they are not btw). Why?